Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/Features/Onboarding/GET.feature

```diff
@@ -0,0 +1,22 @@
+@Integration
+Feature: GET /resource/resourceId?appName=appName#secretKey endpoint
+
+User requests onboarding page
+
+    Scenario: User agent is "iPhone" and only one app-store link should be contained on the response page
+        Given a http client with user agent iPhone
+        When a call is made to the /resource/resourceId endpoint for the enmeshed app
+        Then the response contains a link containing apps.apple.com
+        And the response does not contain a link containing play.google.com
```

Comment:
Since you're always checking for domain names here, I would make the description more precise:
```suggestion
        Then the response contains a link with the domain name "apps.apple.com"
        And the response does not contain a link with the domain name "play.google.com"
```